cli/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ _Released 11/5/2024 (PENDING)_
 
 **Dependency Updates:**
 
+- Updated `mobx` from `5.15.4` to `6.13.5` and `mobx-react` from `6.1.8` to `9.1.1`. Addresses [#30509](https://github.com/cypress-io/cypress/issues/30509).
```

Comment:
@AtofStryker These don't need to be in the changelog, it's a chore.

Reply:
I figured it was best to keep it just in case someone runs into an issue and isn't sure where it came from. I don't think it fails the verify changelog job but I can remove it if you'd like.